src/interactive/runtime.tsx

```diff
@@ -20,7 +20,18 @@ const getRecordings = (useAuthoredData: boolean, recordedData?: SensorRecording)
   if (!useAuthoredData) {
     recording.data = [];
   } else {
-    recording.max = recording.data.reduce((max, point) => Math.max(max, point[1]), recording.max);
+    let yVals = [];
+
+    for (let i = 0; i < recording.data.length; i++){
+      let currentYValue = recording.data[i][1];
+      yVals.push(currentYValue);
+    }
+
+    const positiveYValues = yVals.filter((yVal) => yVal > 0);
+    const smallestPositiveYValue = Math.min(...positiveYValues);
+
+    // We want the max to be set to the next nearest tick after the greatest value.
+    recording.max = recording.data.reduce((max, point) => Math.max(max, point[1] + smallestPositiveYValue), recording.max);
```

Comment:
This is an interesting approach, I would have never thought of it.  Have you checked it out under a few somewhat pathological cases? (sparse data with both positive and negative values comes to mind)

Reply:
Hey Noah, I actually changed this to just be +1. Looking back at this I realized a lot could go wrong with this approach!